src/components/Card/card.js

```diff
@@ -2,14 +2,40 @@ import React from 'react';
 import './card.css';
 
 class Card extends React.Component {
-   
 
+    constructor(props) {
+        super(props);
+        this.state = {
+            value:'!'
+        };
+    }
+
+    
+    checkFlip(id,flip,value) {
+        // console.log("child   "+id)
+            if (flip == false) {
+                this.card.style.backgroundColor = 'rgb(211, 154, 49)';
```

Comment:
instead of doing direct dom manipulation, we can provide classes based on which styles can be applied.